lib/rubocop/cop/rails/render_plain_text.rb

```diff
@@ -53,9 +53,12 @@ def find_content_type(node)
           node.pairs.find { |p| p.key.value.to_sym == :content_type }
         end
 
-        def compatible_content_type?(node)
-          (node && node.value.value == 'text/plain') ||
-            (!node && !cop_config['ContentTypeCompatibility'])
+        def compatible_content_type?(pair_node)
+          if pair_node&.value.respond_to?(:value)
+            pair_node.value.value == 'text/plain'
+          elsif !pair_node
+            !cop_config['ContentTypeCompatibility']
+          end
```

Comment:
Can you tweak to the following?

```suggestion
          if pair_node.nil?
            !cop_config['ContentTypeCompatibility']
          elsif pair_node.value.respond_to?(:value)
            pair_node.value.value == 'text/plain'
          end
```